### ISSUE TITLE
Error while freezing a graph

### ISSUE THREAD

**Issue by jimenisimo** [tensorflow/tensorflow#21683]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Linux Ubuntu 16.04
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**:No
- **TensorFlow installed from (source or binary)**: source
- **TensorFlow version (use command below)**:1.10
- **Python version**:3.6
- **Bazel version (if compiling from source)**:0.16.0
- **GCC/Compiler version (if compiling from source)**:
- **CUDA/cuDNN version**:
- **GPU model and memory**:
- **Exact command to reproduce**:bazel-bin/tensorflow/python/tools/freeze_graph \ --input_graph= /home/my-model/premade_model105/graph.pbtxt \ --input_checkpoint = /home/my-model/premade_model105/model.ckpt-1000.meta \ --output_graph=/home/my-model/premade_model105/frozen_graph.pb \ --output_node_names= dnn/logits/BiasAdd\ --input_binary=false 

### Describe the problem
I am trying to freeze a graph an filling the parameter input_checkpoint with a valid path but I am still getting this error and following the instructions of the official page:

_The input_checkpoint should be the most recent saved checkpoint. As mentioned in the checkpoint section, you need to give the common prefix to the set of checkpoints here, rather than a full filename._

but Iam getting this error:

### **Input checkpoint '' doesn't exist!**

I am using a DNN premade classifier

**Comment by jimenisimo** [comment]
I don't know why the parameter is not being identify, instead it is read as an empty string

**Comment by suharshs** [comment]
You have spaces before and after the `=` for some of the flags. For example `--input_checkpoint = /home/my-model/premade_model105/model.ckpt-1000.meta` should be `--input_checkpoint=/home/my-model/premade_model105/model.ckpt-1000.meta`

Make sure do you that for the other flags and this error should disappear. 

Please reopen if that doesn't resolve it. Thanks!
